cg/apps/demultiplex/demultiplex_api.py

```diff
@@ -42,6 +43,22 @@ def slurm_quality_of_service(self) -> Literal[SlurmQos.HIGH, SlurmQos.LOW]:
         """Return SLURM quality of service."""
         return SlurmQos.LOW if self.environment == "stage" else SlurmQos.HIGH
 
+    @property
```

Comment:
Seems like this decorator is only allowed for functions which do not take an argument: https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=2317&id=Clinical-Genomics_cg&open=AYnf4i84u6bD5Vv_iSlx